src/main/java/org/m14n/moonthespire/PhaseSystem.java

```diff
@@ -57,6 +69,31 @@ private static void modifyDexterity(int amt) {
         );
     }
 
+    public static boolean isWaxing() {
+        return isWaxing;
+    }
+
+    public static boolean isWaning() {
+        return isWaning;
+    }
+
+    public static boolean wasWaxing() {
+        return wasWaxing;
+    }
+
+    public static boolean wasWaning() {
+        return wasWaning;
+    }
+
+    public static void subscribe(PhaseChangeSubscriber subscriber) {
+        subscribers.add(subscriber);
+    }
+
+    public static void unsubscribe(PhaseChangeSubscriber subscriber) {
+        subscribers.remove(subscriber);
+    }
+
+
     static void reset() {

```

Comment:
Oops, we missed this when we were updating PhaseSystem. Could you reset wasWaxing and wasWaning too, here?

Reply:
Sure thing, wasWaxing and wasWaning now reset to False.